DESCRIPTION

```diff
@@ -14,12 +14,14 @@ Authors@R: c(
            email = "jonthegeek@gmail.com",
            comment = c(ORCID = "0000-0003-4781-4346"))
     )
-Description: Tokenize text using a wordpiece vocabulary.
+Description: Apply 'Wordpiece' (<arXiv:1609.08144>) tokenization to input text, 
+ given an appropriate vocabulary. The 'BERT' (<arXiv:1810.04805>) tokenization 
+ conventions are used by default.
 Encoding: UTF-8
 LazyData: true
 URL: https://github.com/jonathanbratt/wordpiece
 BugReports: https://github.com/jonathanbratt/wordpiece/issues
-Depends: R (>= 3.5.1)
+Depends: R (>= 3.5.0)
```

Comment:
Do we actually depend on R 3.5.0+? We should try as well as we can to figure out actual minimum dependencies (this and packages), and likely remove them if we aren't sure. Our internal policy is to be strict, but I recommend we start loose to keep CRAN happy. If we get a bug report that we trace to a package version we can go from there.